ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -157,6 +177,89 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
       </div>;
     };
 
+    function renderInstitutionalAffiliationComponents(componentHandle: React.Component): ReactFragment {
+      if (requireInstitutionalVerification) {
+        return renderVerifiedInstitutionalAffiliationComponents();
+      } else {
+        return renderOldInstitutionalAffiliationComponents(componentHandle);
+      }
+    }
+
+    function renderVerifiedInstitutionalAffiliationComponents(): ReactFragment {
```

Comment:
This looks like a React function component - may want to try just formalizing it as such and see how that compares:
https://fettblog.eu/typescript-react/components/

If you do that, for small cases like this I would probably inline the prop type rather than defining an interface as in the above code example.

Reply:
I have been unable to get that to work

Reply:
Here's an example of it being used in our codebase: https://github.com/all-of-us/workbench/blob/dfc7f2a507fe5128b992f73079ada7d494ef81ed/ui/src/app/icons/card-menu-icon.tsx#L4

Reply:
Hmm, perhaps a bad example since it isn't actually referenced from anywhere.. Presumably it worked at some point..

Reply:
I just remembered this discussion - I wound up using a function component in a recent PR while you were out: https://github.com/all-of-us/workbench/blob/master/ui/src/app/pages/workspace/research-purpose.tsx#L57